lib/rubocop/cop/style/redundant_current_directory_in_path.rb

```diff
@@ -22,16 +22,23 @@ class RedundantCurrentDirectoryInPath < Base
 
         def on_send(node)
           return unless node.method?(:require_relative)
-          return unless node.first_argument.str_content&.start_with?(CURRENT_DIRECTORY_PATH)
+          return unless node.first_argument&.str_content&.start_with?(CURRENT_DIRECTORY_PATH)
           return unless (index = node.first_argument.source.index(CURRENT_DIRECTORY_PATH))
 
-          begin_pos = node.first_argument.source_range.begin.begin_pos + index
-          range = range_between(begin_pos, begin_pos + 2)
+          range = offense_range(node, index: index)
 
           add_offense(range) do |corrector|
             corrector.remove(range)
           end
         end
+
+        private
+
+        def offense_range(node, index:)
+          begin_pos = node.first_argument.source_range.begin.begin_pos + index
+
+          range_between(begin_pos, begin_pos + 2)
+        end
```

Comment:
Can you modify it as below instead of creating a private method?

```diff
--- a/lib/rubocop/cop/style/redundant_current_directory_in_path.rb
+++ b/lib/rubocop/cop/style/redundant_current_directory_in_path.rb
@@ -22,10 +22,11 @@ module RuboCop

         def on_send(node)
           return unless node.method?(:require_relative)
-          return unless node.first_argument.str_content&.start_with?(CURRENT_DIRECTORY_PATH)
-          return unless (index = node.first_argument.source.index(CURRENT_DIRECTORY_PATH))
+          return unless (first_argument = node.first_argument)
+          return unless first_argument.value.start_with?(CURRENT_DIRECTORY_PATH)
+          return unless (index = first_argument.source.index(CURRENT_DIRECTORY_PATH))

-          begin_pos = node.first_argument.source_range.begin.begin_pos + index
+          begin_pos = first_argument.source_range.begin.begin_pos + index
           range = range_between(begin_pos, begin_pos + 2)

           add_offense(range) do |corrector|
```

Reply:
Sure, done